model/client.go

```diff
@@ -25,14 +25,24 @@ import (
 
 // Client holds information about the client of a request.
 type Client struct {
+	// Domain holds the client's domain (FQDN).
+	Domain string
+
 	// IP holds the client's IP address.
 	IP net.IP
+
+	// Port holds the client's IP port.
+	Port int
```

Comment:
Shouldn't the new fields be indexed?

Reply:
Indeed. I was thinking it would be automatic because they're ECS fields

Reply:
added in ba585d166f1393fb07a68b91f3aaf0b6448fecad